src/Driver/WpcliDriver.php

```diff
@@ -111,25 +123,19 @@ public function wpcli($command, $subcommand, $raw_arguments = [])
             $config = sprintf('--path=%s --url=%s', escapeshellarg($this->path), escapeshellarg($this->url));
         }
 
-        // Support Windows.
-        if (DIRECTORY_SEPARATOR === '\\') {
-            $binary = 'wp.bat';
-        } else {
-            $binary = 'wp';
-        }
-
-        // Adjust bootstrap path, dependant on if WordHat is a Composer dependancy, or being run directly.
-        $package = '/vendor/paulgibbs/behat-wordpress-extension';
-        if ($package === substr(getcwd(), -strlen($package), strlen($package))) {
-            $package = '';
-        }
+        //Absolute path to the WordHat src directory
```

Comment:
I'm sorry and I'm trying to not code-review people's code comments, but for comments like this, please can we have a space after the `//` at the start, before the message? My OCD will thank you. :)